.github/workflows/spelling.yml

```diff
@@ -9,7 +9,6 @@ on:
       - "!**/*.min.js"
       - "!**/*.pack.js"
       - "!**/*.paint"
-      - "!aria-practices-DeletedSectionsArchive.html"
       - "!examples/landmarks/css/bootstrap.css"
```

Comment:
Maybe this is covered in another PR/task and doesn't block this PR, but seems this should also be updated to `"!content/patterns/landmarks/examples/css/bootstrap.css"`.